CLIENT-SPECIFICATION.md

```diff
@@ -149,6 +145,8 @@ Steps #3 and #4 may be done in either order.
 
 It is possible that due to this page resolution logic, the client may show a page which does not belong to the host platform because a page can reside in `common`, and not be present on the host platform. Clients must not assume that a given command is always executable on the host platform.
 
+It is suggested for clients to detect new platforms added to `pages` directory automatically and list it as a supported platform in `--platform` option.
```

Comment:
```suggestion
It is suggested for clients to detect new platforms added to `pages` directory automatically and list them as supported platforms in `--platform` option.
```

Reply:
> Where do you think it should be listed?

Probably in the command's usage

Reply:
> Probably in the command's usage

I don't think it's a good idea to try to include supported platforms in the `--help` message. Detecting platforms depends on the cache being available, which is not the case right after installing a client. This means clients have to download pages before they can even show the help message (which should be available offline no matter what).

Reply:
Removed it